grouper/background.py

```diff
@@ -34,19 +35,20 @@ def capture_exception(self):
 
     def run(self):
         while True:
-            logging.debug("Sending emails...")
             try:
                 session = Session()
+                logging.debug("Sending emails...")
                 process_async_emails(self.settings, session, datetime.utcnow())
+                logging.debug("Expiring edges....")
+                expire_edges(session)
+                logging.debug("Pruning old traces....")
                 prune_old_traces(session)
```

Comment:
this is maybe a separate change but i'm wondering if we should be `stats.set_gauge` around each of the operations here (async emails, expiring, and pruning old perf traces). then we can alert on it similar to how we alert on db refresh and config refresh now.